AutoCollection/Statsbeat.ts

```diff
@@ -208,16 +208,27 @@ class Statsbeat {
     }
 
     private _getNetworkStatsbeatCounter(endpoint: number, host: string): Network.NetworkStatsbeat {
+        let shortHost = host;
+        try {
+            let hostRegex = new RegExp(/^https?:\/\/(?:www\\.)?([^\/.]+)/);
```

Comment:
This looks like you have an extra \\, probably should be 
`/^https?:\/\/(?:www\.)?([^\/.]+)/`

Reply:
java version: `^https?://(?:www\.)?([^/.-]+)`
it's missing a dash at the end [^\/.-]

westus02-1 or westus02-4 => send westus02 and we don't care about -1 or -4.